File: src/my_robot_tutorials/scripts/my_first_node.py
#!/usr/bin/env python 
# using python3 instead for noetic

import rospy

if __name__ == '__main__':
	rospy.init_node('my_first_python_node')
	rospy.loginfo("This node has been started")
	#rospy.sleep(1)
	#rospy.loginfo("Exit now")

	rate = rospy.Rate(10) # 10 times per second

	while not rospy.is_shutdown():
		rospy.loginfo("Hello")
		rate.sleep()

File: src/my_robot_tutorials/src/my_first_node.cpp
#include <ros/ros.h>

int main (int argc, char **argv)
{
	ros::init(argc,argv,"my_first_cpp_node");
	ros::NodeHandle nh; // start the node
	ROS_INFO("Node has been started");

	//ros::Duration(1.0).sleep(); // sleep for 1 sec
	//ROS_INFO("Exit");

	ros::Rate rate(10);

	while(ros::ok()){
		ROS_INFO("Hello");
		rate.sleep();
	}
}
